src/tile/tile.scss

```diff
@@ -56,6 +53,20 @@
     }
   }
 
+  &[tabindex] {
+    cursor: pointer;
+    transition: box-shadow $iui-speed-fast ease-in-out;
+
+    &,
+    * {
+      user-select: none;
```

Comment:
Pls remove it, now it is impossible to select any text from the Tile.

Reply:
I added this only to the clickable tiles to avoid accidentally selecting text when clicking on a tile.  My thought was if the entirety of the tile can not be clicked then the text should be selectable.  Do you still think this should be removed?

Reply:
Removed.